.github/workflows/nb_controller_docker_publish.yaml

```diff
@@ -15,30 +15,52 @@ on:
       - components/notebook-controller/**
       - components/common/**
 
+env:
+  DOCKER_USER: kubeflownotebookswg
+  IMG: kubeflownotebookswg/notebook-controller
+  ARCH: linux/ppc64le,linux/amd64
+
 jobs:
   push_to_registry:
     name: Build & Push Docker image to Docker Hub
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@v3
-
-    - name: Login to DockerHub
-      if: github.event_name == 'push'
-      uses: docker/login-action@v2
-      with:
-        username: kubeflownotebookswg
-        password: ${{ secrets.KUBEFLOWNOTEBOOKSWG_DOCKER_TOKEN }}
-
-    - name: Run Notebook Controller build
-      run: |
-        cd components/notebook-controller
-        export IMG=kubeflownotebookswg/notebook-controller
-        make docker-build
-
-    - name: Run Notebook Controller push
-      if: github.event_name == 'push'
-      run: |
-        cd components/notebook-controller
-        export IMG=kubeflownotebookswg/notebook-controller
-        make docker-push
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to DockerHub
+        if: github.event_name == 'push'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ env.DOCKER_USER }}
+          password: ${{ secrets.KUBEFLOWNOTEBOOKSWG_DOCKER_TOKEN }}
+
+      - name: Run Notebook Controller build
+        run: |
+          cd components/notebook-controller
+          make docker-build
+
+      - name: Run Notebook Controller push
+        if: github.event_name == 'push'
+        run: |
+          cd components/notebook-controller
+          make docker-push
```

Comment:
* will drop the docker-build step as it's not required anymore 
*  Docker-push needs to be dropped as it can't be used for pushing multi-arch docker images
*  the "build-multi-arch" rule builds multi-arch  docker images and stores it in build cache 
*the built docker images aren't accessible using "docker image " command, so the built docker images can't be tagged or pushed.

Reply:
I see, thanks for the context! Let's then keep the two steps you had at the end for building, and then building-pushing the image. And drop the `Run Notebook Controller build` and `Run Notebook Controller push` steps